examples/interconnect.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+	"github.com/HewlettPackard/oneview-golang/ov"
+	"os"
+)
+
+func main() {
+	var (
+		clientOV *ov.OVClient
+	)
+	ovc := clientOV.NewOVClient(
+		os.Getenv("ONEVIEW_OV_USER"),
+		os.Getenv("ONEVIEW_OV_PASSWORD"),
+		os.Getenv("ONEVIEW_OV_DOMAIN"),
+		os.Getenv("ONEVIEW_OV_ENDPOINT"),
+		false,
+		600)
+
+	interconnect_list, err := ovc.GetInterconnects("", "", "", "")
```

Comment:
you can add some examples for getting interconnects using filter or other params.

Reply:
please check now.